install/installer/pkg/components/server/types.go

```diff
@@ -33,6 +33,7 @@ type ConfigSerialized struct {
 	VSXRegistryUrl                    string   `json:"vsxRegistryUrl"`
 	ChargebeeProviderOptionsFile      string   `json:"chargebeeProviderOptionsFile"`
 	StripeSettingsFile                string   `json:"stripeSettingsFile"`
+	StripeSecretsFile                 string   `json:"stripeSecretsFile"`
```

Comment:
Shouldn't the line above be removed?

```suggestion
	StripeSecretsFile                 string   `json:"stripeSecretsFile"`
```

Reply:
Yes, I think you are right. It will probably come back again in another PR but let's remove it for now.

[7da5eab](https://github.com/gitpod-io/gitpod/pull/10563/commits/7da5eab26a80b3a98ab3269ae3a49528919d5811)